tls/s2n_connection.h

```diff
@@ -31,10 +31,12 @@
 #include "crypto/s2n_hash.h"
 #include "crypto/s2n_hmac.h"
 
+#include "utils/s2n_blob.h"
 #include "utils/s2n_timer.h"
 #include "utils/s2n_mem.h"
 
 #define S2N_TLS_PROTOCOL_VERSION_LEN    2
+#define S2N_MAX_CERT_TYPE_PREFERENCE_LEN   16
```

Comment:
Nice catch. I added then removed this code in a later commit. Seems I missed a line.